app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module V0
+  class InProgressFormsController < ApplicationController
+    def show
+      form = InProgressForm.where(form_id: params[:id], user_uuid: @current_user.uuid).first
```

Comment:
the form is 'upserted' there will only be one saved form per form_id/user_uuid (actually should be .take rather than .first, slightly faster)